internal/pkg/skuba/kubernetes/versions.go

```diff
@@ -174,7 +207,11 @@ func AddonVersionWithAvailableVersions(addon Addon, clusterVersion *version.Vers
 }
 
 func AddonVersionForClusterVersion(addon Addon, clusterVersion *version.Version) *AddonVersion {
-	return AddonVersionWithAvailableVersions(addon, clusterVersion, Versions)
+	return AddonVersionWithAvailableVersions(addon, clusterVersion, supportVersions)
+}
+
+func AddonsVersionForClusterVersion(clusterVersion *version.Version) AddonsVersion {
```

Comment:
I would maybe try to name this function differently `AllAddonVersionsForClusterVersion`, just to distinguish it more from the function defined right above it `AddonVersionForClusterVersion`.

Reply:
Changed!